docs/cases/cases-overview.asciidoc

```diff
@@ -68,3 +69,32 @@ To view a case, click on its name. You can then:
 * Refresh the case to retrieve the latest updates
 
 NOTE: Comments can also contain Markdown syntax and Timeline links.
+
+[float]
+[[cases-lens-visualization]]
+== Add a Lens visualization
+
+beta[]
+
+Add a Lens visualization to your case to portray event and alert data through charts and graphs.
+
+IMPORTANT: Set an absolute time range for your visualization. This ensures your visualization doesn't change over time after you save it to your case and provides important context for other viewers.
```

Comment:
This admonition seems more helpful to include in the procedure below, in the step where the user is actually creating/configuring the visualization.